core/miner/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ add_subdirectory(storage_fsm)
 
 add_library(miner
         impl/miner_impl.cpp
-    )
+            )
```

Comment:
Please, revert or reformat the whole file to be same as the others cmake files.

Reply:
fixed